numba/tests/test_indexing.py

```diff
@@ -521,6 +528,11 @@ def test_1d_integer_indexing(self, flags=enable_pyobj_flags):
         self.assertEqual(pyfunc(a, 9), cfunc(a, 9))
         self.assertEqual(pyfunc(a, -1), cfunc(a, -1))
 
+        # use IntEnumMember as index
+        self.assertEqual(pyfunc(a, 0), cfunc(a, EnumIndex.INDEX0))
```

Comment:
These tests are testing with a function compiled for argument types `(arraytype, types.int32)` only - therefore it's not actually testing with an `IntEnumMember`. You need to compile a new version where the second argument is of the `IntEnumMember` type (e.g. `types.IntEnumMember(EnumIndex, types.int32)`) to test using an `IntEnumMember` as an array index.

Reply:
rewrote the test cases. Thanks for ur suggestions!